### ISSUE TITLE
Window Narrator does not pronounce title

### ISSUE THREAD

**Issue by skfd** [nathanboktae/aria-listbox#2]
In your [example](http://nathanboktae.github.io/aria-listbox/) Narrator does not prounounce `listbox` title "Choose a Fruit" when you tab into it.

**Comment by nathanboktae** [comment]
Yes I should have it use [`aria-labeledby`](https://www.w3.org/TR/2010/WD-wai-aria-20100916/states_and_properties#aria-labelledby) in that example. Not a bug of this library, but I can update the example. Thanks.
